src/validation_workflow/api_request.py

```diff
@@ -20,15 +19,11 @@ class ApiTest:
 
     def __init__(self, request_url: str) -> None:
         self.request_url = request_url
-        self.apiHeaders_auth = {"Authorization": "Basic YWRtaW46YWRtaW4="}  # default user/pass "admin/admin" in Base64 format
-        self.apiHeaders_accept = {"Accept": "*/*"}
-        self.apiHeaders_content_type = {"Content-Type": "application/json"}
-        self.apiHeaders = {}
-        self.apiHeaders.update(self.apiHeaders_auth)
-        self.apiHeaders.update(self.apiHeaders_accept)
-        self.apiHeaders.update(self.apiHeaders_content_type)
 
     def api_get(self) -> Any:
-
-        response = requests.get(self.request_url, headers=self.apiHeaders, verify=False)
-        return response.status_code, response.text
+        self.command = ['curl', self.request_url, '-u', 'admin:admin', '--insecure', '-i']
```

Comment:
Why are we replacing a native HTTP implementation with `curl`?

Reply:
Thanks for the comment, @dblock .. a few weeks back, when I was doing testing with @peterzhuamazon , we found out that the Curl is more stable on privding the result than using the request lib for some reasons.

Reply:
Thanks, dB, I will keep using the requests libarary. so, no change on this script.